### PR TITLE
fix support for black 22.1

### PR DESCRIPTION
**Pull request by jabesq**

### CODE REVIEW

File: src/darker/config.py [hunk 1]
```diff
@@ -104,7 +104,8 @@ def load_config(srcs: Iterable[str]) -> DarkerConfig:
                  are used to look for the ``pyproject.toml`` configuration file.
 
     """
-    path = find_project_root(tuple(srcs or ["."])) / "pyproject.toml"
+    project_root = find_project_root(tuple(srcs or ["."]))[0]
+    path = project_root / "pyproject.toml"
```
Comment (Mystic-Mirage) [inline src/darker/config.py:108]: Let's use `find_pyproject_toml` instead:

```python
    path = find_pyproject_toml(tuple(srcs or ["."]))
    if path is not None:
```

Comment (akaihola) [reply src/darker/config.py:108]: What if someone uses Darker with a Python code base which doesn't have `pyproject.toml`?

**Comment by akaihola** [comment]
As noted elsewhere, in #270 I took the approach of supporting both the old and the new Black behavior.

**Comment by akaihola** [comment]
#270 has been merged instead. Thank you @jabesq for your contribution nevertheless!

**Pull request closed without merging**